packages/api-client/src/conversation/ConversationAPI/ConversationAPI.ts

```diff
@@ -856,11 +856,11 @@ export class ConversationAPI {
    * @param typingData The typing status
    * @see https://staging-nginz-https.zinfra.io/swagger-ui/#!/conversations/isTyping
    */
-  public async postTyping(conversationId: string, typingData: ConversationTypingData): Promise<void> {
+  public async postTyping(conversationId: QualifiedId, typingData: ConversationTypingData): Promise<void> {
     const config: AxiosRequestConfig = {
       data: typingData,
       method: 'post',
-      url: `${ConversationAPI.URL.CONVERSATIONS}/${conversationId}/${ConversationAPI.URL.TYPING}`,
+      url: `${this.generateBaseConversationUrl(conversationId)}/${ConversationAPI.URL.TYPING}`,
```

Comment:
Wait, I'm not sure this endpoint will work with api v2?

Reply:
Ok, seems to be fine with backend, the endpoint is going to be available for v2 as well, all good then 👌

Reply:
Yes, we should be in sync with what's documented in swagger 👍